NSS-Day-Cohort-42/rare-media-moguls
# Cb register

#### Changes Made
1. Modified file `Register.js` to allow user to save their information to database when they click register button
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout cb-register
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
> When user enters information into the form fields and clicks `Save`, the post database should be updated with new user object 

6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
